### ISSUE TITLE
Configuration shows old price when chosen, although the price is right in basket.

### ISSUE THREAD

**Issue by fouli86** [magento/magento2#12778]
### Preconditions

1. Magento CE 2.1.11
2. PHP 7.0.22
3. Server version: MySQL 5.6.35

### Steps to reproduce

1. Create Configurable product and add simple products as configurations
2. Set a discount to the entire product, both parent product and configurations
3. Try order the product from the frontend by choosing the configuration you want.

### Expected result

1. The configuration must show the new price with the discount, when chosen in order to add it to the basket.

### Actual result

The configuration shows only the old price but when added to basket, the price is the right one. You can then proceed to the order without any problems. I have flushed the cache multiple times, I have reindexed, checked cron, upgraded Magento from 2.1.10 to 2.1.11, but still nothing.

**Comment by magento-engcom-team** [comment]
@fouli86 , thank you for your report.
When you create Catalog Price Rule, the discount is applied on Product page. Cart price Rule is applying and show discount only on Checkout page. Please check this.

**Comment by fouli86** [comment]
Thank you for your reply. There is neither Catalog Price Rule nor Cart price Rule applied at the moment. 
I have the same Special Price on both parent product and configurations with the same starting date and no ending one. On the Product Page, the special price is visible, but once the customer clicks on the configuration the special price disappears and only the initial one is visible. Once placed the configuration in the cart the special price reappears and remains for the rest of the checkout process. 

In other words, the Special Price is not visible only during the initial selection of the Configuration from the Product Page.

I have also tried creating a new order from the backend and everything worked fine. For me, it is a matter of visibility on the product page view, regarding only the configurations. 

Thanks is advance

**Comment by magento-engcom-team** [comment]
@fouli86, thank you for your answer.
Unfortunately, we cannot reproduce this issue. Please provide more detailed steps to reproduce.

**Comment by magento-engcom-team** [comment]
@fouli86, we are closing this issue due to inactivity. If you'd like to update it, please reopen the issue.
